### DISCUSSION TITLE
馬斯克擬允解除全自動駕駛系統警示

### DISCUSSION THREAD

**Discussion by Yilin20021016** [1101-GS3073-C/Python-Community-GS3073-C#679] [General]
感覺是一個可怕的東西 萬一出bug了怎麼做
https://technews.tw/2023/01/10/musk-appears-to-agree-to-unlock-full-self-driving-system/

**Comment by Aerofe** [comment]
全自動駕駛的情況下駕駛者還是得注意路況，以免發生複雜事件時，AI無法處理而導致事故發生

**Comment by Valentinaa0214** [comment]
目前的技術感覺還不是那麼成熟，全自動駕駛系統意味著幾乎是把自己的性命交給機器人的感覺，是我的話其實有點不敢。
要是真的出現AI無法判斷的情況，可能真的會導致嚴重事故的發生。

**Comment by HsuChen88** [comment]
當汽車是由某一個集中的系統在操控時，那只要駭入這個系統，就可以向電影中，由一個人操控所有的車子，造成交通大亂。

**Comment by yjpxi** [comment]
如果消費者認為特斯拉的全自動駕駛系統不夠安全，這可能會削弱對自動駕駛技術的信心，導致購買這類車輛的意願下降。
